config/database.yml

```diff
@@ -20,7 +20,7 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  port: 5433
+  port: 5432
```

Comment:
[Optional]: This should be Ignored, It looks like you have pushed this file before adding it to ignore file

Reply:
I did a mistake when I added database.yml on .gitignore but I fixed it.